### PR TITLE
J-123: + create users_notifications migration

### PR DESCRIPTION
**Pull request by artem-totality**

### CODE REVIEW

File: backend/src/data/migrations/20210811124705_add_users_notifications_table.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Knex } from 'knex';
```
Comment (ghost) [inline backend/src/data/migrations/20210811124705_add_users_notifications_table.ts:1]: don't forget about model, types and etc

File: backend/src/data/migrations/20210811124705_add_users_notifications_table.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const TABLE_NAME = 'users_notifications';
+
+export async function up(knex: Knex): Promise<void> {
+  const dateNowISO = getFormattedISODate(new Date());
+
+  return knex.schema.createTable(TABLE_NAME, (table) => {
+    table.increments('id').primary();
+    table.integer('user_id').notNullable().references('id').inTable('users');
+    table.integer('notification_id').notNullable().references('id').inTable('notifications');
+    table.boolean('is_checked').notNullable().defaultTo(false);
```
Comment (ghost) [inline backend/src/data/migrations/20210811124705_add_users_notifications_table.ts]: we have enum here? 🤔

**Review by ghost — CHANGES_REQUESTED** [review]

File: backend/src/data/migrations/20210811124705_add_users_notifications_table.ts (hunk 2 above)
Comment (artem-totality) [reply backend/src/data/migrations/20210811124705_add_users_notifications_table.ts]: No, we have not used any enums before

Comment (ghost) [reply backend/src/data/migrations/20210811124705_add_users_notifications_table.ts]: We don't have any flags in this entity
image
check your task

**Review by ghost — CHANGES_REQUESTED** [review]

File: backend/src/data/migrations/20210811124705_add_users_notifications_table.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { Knex } from 'knex';
+import { getFormattedISODate } from '~/helpers/helpers';
+import { UserNotificationStatus } from '~/common/enums/enums';
```
Comment (ghost) [inline backend/src/data/migrations/20210811124705_add_users_notifications_table.ts]:
```suggestion
```
create a new enum inside migration

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**